ui/apps/platform/src/Containers/Policies/Wizard/PolicyWizard.tsx

```diff
@@ -149,73 +150,105 @@ function PolicyWizard({ pageAction, policy }: PolicyWizardProps): ReactElement {
+                                    footer={{ isNextDisabled: !isValidOnClient }}
+                                >
                                     <PolicyDetailsForm
                                         id={values.id}
                                         mitreVectorsLocked={values.mitreVectorsLocked}
                                     />
-                                ),
-                                canJumpTo: canJumpToAny || stepIdReached >= 1,
-                                enableNext: isValidOnClient,
-                            },
-                            {
-                                id: 2,
-                                name: 'Policy behavior',
-                                component: (
+                                </WizardStep>,
+                                <WizardStep
+                                    name="Lifecycle"
+                                    id={POLICY_DEFINITION_LIFECYCLE_ID}
+                                    key={POLICY_DEFINITION_LIFECYCLE_ID}
+                                    body={{ hasNoPadding: true }}
+                                    footer={{ isNextDisabled: !isValidOnClient }}
+                                >
                                     <PolicyBehaviorForm hasActiveViolations={hasActiveViolations} />
-                                ),
-                                canJumpTo: canJumpToAny || stepIdReached >= 2,
-                                enableNext: isValidOnClient,
-                            },
-                            {
-                                id: 3,
-                                name: 'Policy criteria',
-                                component: (
+                                </WizardStep>,
+                                <WizardStep
+                                    name="Rules"
+                                    id={POLICY_DEFINITION_RULES_ID}
+                                    key={POLICY_DEFINITION_RULES_ID}
+                                    body={{ hasNoPadding: true }}
+                                    footer={{ isNextDisabled: !isValidOnClient }}
+                                >
                                     <PolicyCriteriaForm hasActiveViolations={hasActiveViolations} />
-                                ),
-                                canJumpTo: canJumpToAny || stepIdReached >= 3,
-                                enableNext: isValidOnClient,
-                            },
-                            {
-                                id: 4,
-                                name: 'Policy scope',
-                                component: <PolicyScopeForm />,
-                                canJumpTo: canJumpToAny || stepIdReached >= 4,
-                                enableNext: isValidOnClient,
-                            },
-                            {
-                                id: 5,
-                                name: 'Review policy',
```

Comment:
Integration test failure because need to change selector here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/constants/PoliciesPage.js#L35

> Expected to find element: `.pf-v5-c-wizard__nav-link:contains("Review policy")`, but never found it.

Reply:
Unlike other steps for which `name` prop is consistent with `Title` element in component:
* name: Review
* Title: Review policy